src/plugin/cursus/Resources/modules/data/sources/courses.js

```diff
@@ -0,0 +1,28 @@
+import {trans} from '#/main/app/intl/translation'
+import {URL_BUTTON} from '#/main/app/buttons'
+
+import {route as toolRoute} from '#/main/core/tool/routing'
+
+import {route} from '#/plugin/cursus/routing'
+import {CourseCard} from '#/plugin/cursus/course/components/card'
+
+export default {
+  name: 'courses',
+  icon: 'fa fa-fw fa-cubes',
+  parameters: {
+    primaryAction: (course) => ({
+      type: URL_BUTTON,
+      target: `#${route(toolRoute('trainings')+'/catalog', course)}`
+    }),
+    definition: [
```

Comment:
Fill the definition with the same columns than : https://github.com/claroline/Claroline/blob/13.x/src/plugin/cursus/Resources/modules/course/components/list.jsx#L31.

Reply:
I'm filled definition with all columns